### DISCUSSION TITLE
Android auto always shows download map screen

### DISCUSSION THREAD

**Discussion by trolles** [organicmaps/organicmaps#7624] [Q&A]
Hey there, I'm quite happy using OM through AAWireless and Android Auto but there is always a minor inconvenience when connecting it. 

The screen to download map always shows at startup on the car and I need to open the app on the phone, then click on car screen to get control and close app on the phone. If not, the car screen shows the currently downloading screen but never finishes. 

I have tried to delete cache and data on the phone then allowing the download from the car, which works fine until I connect it again for another ride. 

How can I avoid that screen showing on the car?

![IMG_20240318_073944_664.jpg](https://github.com/organicmaps/organicmaps/assets/10604261/018051ad-e301-45ee-bb5b-65588bf4e1f9)

**Comment by biodranik** [comment]
Did you download all maps before in OM? Did it work on a phone?

**Reply by trolles** [reply]
Phone is working without issue. I have only downloaded the world map and my local region. I will download all my country maps and try again.

**Reply by trolles** [reply]
Same result. 

I get "See update" screen, then another screen warning about the download and then the downloading that never finishes. See pictures attached. 

![IMG_20240318_170628_689.jpg](https://github.com/organicmaps/organicmaps/assets/10604261/6a6594c5-7893-4005-bcb8-23044ca3ea2a)

![IMG_20240318_170619_809.jpg](https://github.com/organicmaps/organicmaps/assets/10604261/00f80f52-2d46-4c2d-ae7c-f53f51947e6c)

![IMG_20240318_170612_442.jpg](https://github.com/organicmaps/organicmaps/assets/10604261/f595402b-12d6-405f-bdd0-47089c3e7d6c)

**Reply by biodranik** [reply]
Are you using fdroid or web builds of Organic Maps? Can you please confirm again that World and WorldCoasts, and your local map regions are downloaded and working in the phone without connecting it to AA, so it is reproduced only in AA?

**Reply by trolles** [reply]
Yes, all those maps work fine on the phone, at least I don't get any message at start up. 

I'm using version 2024.03.05-4-FDroid at the moment.
